### ISSUE TITLE
Symfony `cache:clear` + `cache:warmup` does not always populate Ecotone cache directory

### ISSUE THREAD

**Issue by someniatko** [ecotoneframework/ecotone-dev#217]
**Ecotone version(s) affected**: 1.104.1

**Description** 
In default configuration of the ecotone/symfony-bundle and Symfony itself, assuming `APP_ENV=prod`, there is a cache directory `<project root>/var/cache/prod/ecotone`. It is created after clearing + warming up the Symfony cache, but not always populated. However, on subsequent cache clears it may be populated.

I warmup the Symfony-related cache during CI builds of my project, therefore it is crucial for me that cache clears work out in a stable manner.

My ultimate goal is to have `var/cache/prod` directory actually immutable and be built only once. However, there is another problem which is it seems Ecotone proxy classes are regenerated on each request (instead of being only generated once when clearing / warming up the cache), but this is another story.

**How to reproduce**  
For my project it is:

1. For the "clean experience", physically remove or empty `var/cache/prod` directory.
2. Execute `php bin/console cache:clear` and `php bin/console cache:warmup`.
3. Look at the `var/cache/prod/ecotone` dir. It will be empty.
4. Execute step 2 again.
5. Look at the `var/cache/prod/ecotone` dir. It will probably populated, but in my CI builds it may actually take 3 cache clears for some reason (the dir is not populated after 2nd time sometimes, idk why actually)

I have not tried it on an clean project though. If you can't reproduce it on your own, I will try my best to come up with a reproducer repository.

**Context**  
Symfony 6.3

**Comment by dgafka** [comment]
Hey, 
There was no hook in for cache warmup, this will be fixed with #218 :) 

> Ecotone proxy classes are regenerated on each request 

In production env, that should also be populated automatically. As Ecotone collects all the [Gateways](https://docs.ecotone.tech/messaging/messaging-concepts/messaging-gateway) (Interface for which, proxies are generated) and dump proxy classes for those.

However, if you're using integration with JMS (`ecotone/jms-converter`), then JMS metadata will be populated on fly. 
To improve this, we would need to iterate over all Command/Query/Event classes in order to generate metadata files.

**Comment by dgafka** [comment]
[1.106.0](https://github.com/ecotoneframework/ecotone-dev/releases/tag/1.106.0)

**Comment by someniatko** [comment]
@dgafka Since 1.107, `cache:warmup` now again does not populate `ecotone` cache dir. It only creates `messaging_system` file, but proxies are generated on fly :(

**Comment by dgafka** [comment]
@someniatko it will work on 1.108.1, have fun :)
